applications/OptimizationApplication/python_scripts/controls/simp_control.py

```diff
@@ -0,0 +1,146 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.OptimizationApplication as KratosOA
+from KratosMultiphysics.OptimizationApplication.controls.control import Control
+from KratosMultiphysics.OptimizationApplication.optimization_info import OptimizationInfo
+from KratosMultiphysics.OptimizationApplication.utilities.response_function_implementor import ResponseFunctionImplementor
+from KratosMultiphysics.OptimizationApplication.utilities.helper_utilities import WriteCollectiveVariableDataHolderToOptmizationInfo
+
+class SimpControl(Control):
```

Comment:
Move the base control class to c++, and change optimization_info to be able to use in C++

Reply:
optimization_info will be able to store `double`, `int`, `string` ,`ContainerVariableData`